src/product.hpp

```diff
@@ -232,6 +232,11 @@
 
 #define SF_UPLOAD_ENCODING SF_UPLOAD_BASE64URL
 
+#define UBLOX_CS_PIN 53
+#define UBLOX_PWR_EN_PIN 41
+#define UBLOX_RESETN_PIN 44
+#define UBLOX_SPI_INTERFACE                   (SPI1)
```

Comment:
Please add docstrings to make clear what these are for.  Also, these are not used anywhere in the code!

Reply:
Added